theme/lumo/vaadin-grid-styles.html

```diff
@@ -88,57 +88,88 @@
         box-shadow: inset 0 0 0 2px var(--lumo-primary-color-50pct);
       }
 
-      [part~="row"][row-drag] {
-        z-index: 100;
+      [part~="row"][dragover] {
+        z-index: 100 !important;
       }
 
-      [part~="row"][row-drag] [part~="cell"] {
+      [part~="row"][dragover] [part~="cell"] {
         overflow: visible;
       }
 
-      [part~="row"][row-drag] [part~="cell"]::before {
+      [part~="row"][dragover] [part~="cell"]::after {
         content: "";
         position: absolute;
-        top: 1px;
+        top: 0;
         right: 0;
-        bottom: 1px;
+        bottom: 0;
         left: 0;
+        height: calc(var(--_lumo-grid-border-width) + 2px);
         pointer-events: none;
+        background: var(--lumo-primary-color-50pct);
       }
 
-      [part~="row"][row-drag="below"] [part~="cell"]::before {
-        box-shadow: 0 3px 0 0 var(--lumo-primary-color-50pct);
+      :host([theme~="no-row-borders"]) [dragover] [part~="cell"]::after {
+        height: 2px;
       }
 
-      [part~="row"][row-drag="above"] [part~="cell"]::before {
-        box-shadow: 0 -3px 0 0 var(--lumo-primary-color-50pct);
+      [part~="row"][dragover="below"] [part~="cell"]::after {
+        top: 100%;
+        bottom: auto;
+        margin-top: -1px;
```

Comment:
**Suggestion:** Should we use `calc` for some lumo variable over here, like `calc(var(--lumo-space-xs) / 4)` ?

Reply:
The fixed sizing comes from the static focus indicator sizing https://github.com/vaadin/vaadin-grid/blob/master/theme/lumo/vaadin-grid-styles.html#L75